src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Color {
+    Blue,
+    Green,
+    Red,
+    Yellow,
+    Purple

```

Comment:
Enum values should be named in uppercase letters. Please rename the enum constants according to the Java style guide.